src/addons/dragAndDrop/EventContainerWrapper.js

```diff
@@ -157,7 +184,11 @@ class EventContainerWrapper extends React.Component {
       const bounds = getBoundsForNode(node)
       const { dragAndDropAction } = this.context.draggable
 
-      if (dragAndDropAction.action === 'move') this.handleMove(box, bounds)
+      if (dragAndDropAction.action === 'move') {
+        if (this.context.draggable.enableDragAutoScroll)
+          this.updateParentScroll(parent, node)
+        this.handleMove(box, bounds)
+      }
       if (dragAndDropAction.action === 'resize') this.handleResize(box, bounds)
```

Comment:
Can you check the experience for scrolling on resize too? When I am resizing and the event is on boundary of my scrollable window, it might make sense to scroll.

Reply:
It is a good idea, I have checked it completely makes sense